bigtable/client/internal/table_admin.cc

```diff
@@ -193,6 +193,42 @@ bool TableAdmin::CheckConsistency(
   return response.consistent();
 }
 
+std::vector<btproto::Snapshot> TableAdmin::ListSnapshots(
+    int32_t page_size, grpc::Status& status,
```

Comment:
We use `std::int32_t`, the fact that protobuf and grpc lazily pollute the user namespace does not mean that we should.